pyccl/baryons/baccoemu_baryons.py

```diff
@@ -127,11 +132,17 @@ def boost_factor(self, cosmo, k, a):
             else:
                 emupars['A_s'] = cosmo['A_s']
 
+        # change masses from Msun to Msun/h
+        _bcm_params = deepcopy(self.bcm_params)
+        for key in ['M_c', 'M1_z0_cen', 'M_inn']:
+            _bcm_params[key] = np.log10((10**_bcm_params[key])
+                                        * emupars['hubble'])
```

Comment:
Annoying query: this is faster if you do simply `_bcm_params[key] += l10h`, with `l10h = np.log10(emupars['hubble'])` precalculated earlier.

Reply:
Cool!